src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -857,7 +857,7 @@ private function registerRouterConfiguration(array $config, ContainerBuilder $co
         $loader->load('routing.xml');
 
         if (null === $config['utf8']) {
-            @trigger_error('Not setting the "framework.router.utf8" configuration option is deprecated since Symfony 5.1, it will default to "true" in Symfony 6.0.', E_USER_DEPRECATED);
+            deprecated('symfony/framework-bundle', 5.1, 'Not setting the "framework.router.utf8" configuration option is deprecated, it will default to "true" in version 6.0.');
```

Comment:
```suggestion
            deprecated('symfony/framework-bundle', '5.1', 'Not setting the "framework.router.utf8" configuration option is deprecated, it will default to "true" in version 6.0.');
```

As mentioned in https://github.com/symfony/symfony/pull/35526#issuecomment-583313973 already, casting float to string is a brittle operation in php. I'd strongly recommend to use strings for version numbers.

Reply:
updated.